From 1cd5bb1e264e364124891688c0dbabdb73d4031a Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@serpentos.com>
Date: Mon, 12 Feb 2024 00:38:04 +0000
Subject: [PATCH 2/2] sysusers/basic: Remove all definitions bar `root`

We now handle all statically allocated system users and groups
via the `userdb` mechanism in the `system-accounts` package in
Serpent OS.

Signed-off-by: Ikey Doherty <ikey@serpentos.com>
---
 sysusers.d/basic.conf.in | 31 -------------------------------
 1 file changed, 31 deletions(-)

diff --git a/sysusers.d/basic.conf.in b/sysusers.d/basic.conf.in
index 0aec080..2bc6354 100644
--- a/sysusers.d/basic.conf.in
+++ b/sysusers.d/basic.conf.in
@@ -8,34 +8,3 @@
 # The superuser
 g root    0       -            -
 u root    0:0     "Super User" /root
-
-# The nobody user/group for NFS file systems
-g {{NOBODY_GROUP_NAME}} 65534       -            -
-u {{NOBODY_USER_NAME }} 65534:65534 "Kernel Overflow User"     -
-
-# Administrator group: can *see* more than normal users
-g adm     {{ADM_GID    }}     -            -
-
-# Administrator group: can *do* more than normal users
-g wheel   {{WHEEL_GID  }}     -            -
-
-# Access to shared database of users on the system
-g utmp    {{UTMP_GID   }}     -            -
-
-# Physical and virtual hardware access groups
-g audio   {{AUDIO_GID  }}     -            -
-g cdrom   {{CDROM_GID  }}     -            -
-g dialout {{DIALOUT_GID}}     -            -
-g disk    {{DISK_GID   }}     -            -
-g input   {{INPUT_GID  }}     -            -
-g kmem    {{KMEM_GID   }}     -            -
-g kvm     {{KVM_GID    }}     -            -
-g lp      {{LP_GID     }}     -            -
-g render  {{RENDER_GID }}     -            -
-g sgx     {{SGX_GID    }}     -            -
-g tape    {{TAPE_GID   }}     -            -
-g tty     {{TTY_GID    }}     -            -
-g video   {{VIDEO_GID  }}     -            -
-
-# Default group for normal users
-g users   {{USERS_GID  }}     -            -
-- 
2.43.0

